src/U_polynomials.jl

```diff
@@ -1,5 +1,5 @@
 function Uk_poly_Jn(p, v, p2, x::T) where T <: Float64
-    if v > 5.0 + 1.00033*x + cbrt(1427.61*x)
+    if v > 5.0 + 1.00033*x + Base.Math._approx_cbrt(1427.61*Float64(x))
```

Comment:
why `Float64(x)`?

Reply:
In the tests I was propagating big floats through some of the float64 functions to make sure they were accurate to ~1e-16 and there wasn’t a BigFloat implementation. It looks like I restrict the type here so actually not needed